package.json

```diff
@@ -12,67 +12,68 @@
   "scripts": {
     "start": "nps",
     "test": "nps test",
-    "precommit": "lint-staged && opt --in pre-commit --exec \"npm start validate\""
+    "prepare": "patch-package && simple-git-hooks"
   },
   "files": [
     "dist"
   ],
   "keywords": [],
   "author": "Kent C. Dodds <kent@doddsfamily.us> (http://kentcdodds.com/)",
   "maintainers": [
-    "Simon Fridlund <simon@fridlund.email>"
+    "Simon Fridlund <simon@fridlund.email>",
+    "JounQin (https://www.1stG.me) <admin@1stg.me>"
   ],
   "license": "MIT",
   "dependencies": {
     "arrify": "^2.0.1",
-    "boolify": "^1.0.0",
-    "camelcase-keys": "^6.0.0",
-    "chalk": "^2.4.2",
-    "common-tags": "^1.8.0",
-    "core-js": "^3.1.4",
-    "eslint": "^8.14.0",
-    "find-up": "^4.1.0",
-    "get-stdin": "^7.0.0",
-    "glob": "^7.1.4",
-    "ignore": "^5.1.2",
+    "boolify": "^1.0.1",
+    "camelcase-keys": "^7.0.2",
+    "chalk": "^4.1.2",
+    "common-tags": "^1.8.2",
+    "core-js": "^3.24.1",
+    "eslint": "^8.21.0",
+    "find-up": "^5.0.0",
+    "get-stdin": "^8.0.0",
+    "glob": "^7.2.3",
+    "ignore": "^5.2.0",
```

Comment:
I thought we were only updating devDependencies in this PR.

Reply:
I'd like to bump all for a new major release.